doc/usecases/polyculture/add_plant_relationships.md

```diff
@@ -0,0 +1,33 @@
+# Use Case: Add Plant Relationships
+
+## Summary
+
+- **Scope:** Plants Layer
+- **Level:** User Goal
+- **Actors:** App User
+- **Brief:** A new plant relationship is added to the system.
+- **Status:** Draft
+
+## Scenarios
+
+- **Precondition:**
+  - The user has the opened the UI to propose a new relationship.
+- **Main success scenario:**
+  - Details about the relationship are provided by the user.
+     - The two sides of the relationship can be
+       - Specific plants
+       - Any higher taxonomic level (e.g. family, subfamily)
+     - The type of the relationship can be
+       - Companion
+       - Antagonist
+     - A description providing information on the whys and hows.
+- **Alternative scenario:**
+  - The user creates a relationship which is only effective for one or more of her selected maps.  
+    In this case there is no review process.
+- **Error scenario:**
+  - A relationship with identical sides already exits.  
+    The user is informed about this and can't create the relationship.
```

Comment:
See above, map-specific overrides are okay.

Reply:
Ok, but the following should be an error too, right?

The user tries to add a map-specific one where a map-specific one with identical sides already exists.

Reply:
Yes, this is an error.